test/updater_unittest.cc

```diff
@@ -192,26 +193,24 @@ TEST_F(UpdaterTest, UpdateMetadataCallback) {
       std::chrono::system_clock::now(),
       json::object({{"f", json::string("test")}}));
   MonitoredResource resource("test_resource", {});
-  const std::string frn = "/test";
   MetadataUpdater::ResourceMetadata metadata(
```

Comment:
[Optional] I'd propose renaming this one to `resource_metadata` and renaming `metadata_list` below to `metadata`, for consistency with other places. But this is optional, so if you choose not to do it, just ack.

Reply:
I just renamed this to mr for monitored resource. Hope this looks okay.

Reply:
Renamed `mr` to `resource_metadata`. `metadata_list` was already `metadata`.